torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -39,6 +39,22 @@ namespace c10d {
 
 constexpr const char* const kNCCLAbortedCommStoreKey = "NCCLABORTEDCOMM";
 
+std::vector<std::string> ENABLE_NCCL_HEALTH_CHECK = {
+    "ENABLE_NCCL_HEALTH_CHECK"};
+std::vector<std::string> NCCL_BLOCKING_WAIT = {"NCCL_BLOCKING_WAIT"};
+std::vector<std::string> NCCL_ASYNC_ERROR_HANDLING = {
+    "NCCL_ASYNC_ERROR_HANDLING"};
+std::vector<std::string> NCCL_DESYNC_DEBUG = {"NCCL_DESYNC_DEBUG"};
+std::vector<std::string> NCCL_ENABLE_TIMING = {"NCCL_ENABLE_TIMING"};
+std::vector<std::string> TORCH_NCCL_ENABLE_MONITORING = {
+    "TORCH_NCCL_ENABLE_MONITORING"};
+std::vector<std::string> TORCH_NCCL_HEARTBEAT_TIMEOUT_SEC = {
+    "TORCH_NCCL_HEARTBEAT_TIMEOUT_SEC"};
+std::vector<std::string> TORCH_NCCL_AVOID_RECORD_STREAMS = {
+    "TORCH_NCCL_AVOID_RECORD_STREAMS"};
+std::vector<std::string> NCCL_USE_TENSOR_REGISTER_ALLOCATOR_HOOK = {
+    "NCCL_USE_TENSOR_REGISTER_ALLOCATOR_HOOK"};
+
 DebugInfoWriter::DebugInfoWriter(int rank) {
   const char* fileName = parseEnvVarString(
```

Comment:
need to replace this as well

Reply:
This patch does not rename the variables yet.  It simply introduces the new getCvar functions.  I have another commit that makes the actual changes to the env variables, which I have not pushed yet.